internal/stage6.go

```diff
@@ -35,20 +36,8 @@ func testType1(stageHarness *test_case_harness.TestCaseHarness) error {
 		}
 	}
 
-	invalidCommands := getRandomInvalidCommands(2)
-
-	for _, invalidCommand := range invalidCommands {
-		command := fmt.Sprintf("type %s", invalidCommand)
-
-		testCase := test_cases.CommandResponseTestCase{
-			Command:          command,
-			ExpectedOutput:   fmt.Sprintf("%s: not found", invalidCommand),
-			FallbackPatterns: []*regexp.Regexp{regexp.MustCompile(fmt.Sprintf(`^(bash: type: )?%s[:]? not found$`, invalidCommand))},
-			SuccessMessage:   "✓ Received expected response",
-		}
-		if err := testCase.Run(asserter, shell, logger); err != nil {
-			return err
-		}
+	if err := invalid_command.TestTypeResponses(asserter, shell, logger, 2); err != nil {
```

Comment:
@andy1li I think we've gone a bit overboard with extracting things into this package - can you try to keep things a bit more close to what we're doing with `CDAndPwdTestCase`? This seems like something that'd neatly fit into 1/2 `SomethingTestCase` structs - don't think we'd need to deviate from that structure. Our primary goal is to avoid inconsistency in the "ExpectedOutput" / "FallbackPatterns" - for that we don't need to extract things like loops. 

Ideas to make this more consistent with the rest of the code: 

- Keep things like looping logic within the stage functions, that doesn't need to be extracted

- Extract 1 or 2 TestCase structs, they can be placed within the existing `test_cases` package. Name ideas: `InvalidCommandTestCase`, `InvalidCommandTypeTestCase`